source/_posts/2025-02-05-release-20252.markdown

```diff
@@ -0,0 +1,539 @@
+then triggers a backup.
+
+### Re-introduction of the backup toggle on update
+
+If you run the Home Assistant OS, you get updates for Home Assistant and
+all add-ons. This used to have a toggle to create a backup before updating,
+which was removed in the previous release.
+
+This change was made because when on running automated regular backups, this
+feature was considered redundant. However, we've received a lot of feedback
+that this feature was used by many, to ensure they have the latest data before
+updating and, in case of add-ons, to have a backup of the previous version
+of the add-on.
+
+Valid cases, and we are re-introducing this backup on update toggle in this
+release, but in a smarter way.
+
+#### Backup on updating Home Assistant
+
+When updating Home Assistant, the toggle is now turned off by default and
+shows the last time you made a backup. This gives you a clear indication of
+when you last made a backup, and gives you the ability to decide if you want
+to make a new one before updating.
+
+<img class="no-shadow" alt="Temporary screenshot" src="/images/blog/2025-02/backup-toggle-update.png" />
+
+If you turn the switch on, a full automated backup is created and synced to
+your configured locations, before the update is started.
+
+This full sync is done to ensure you have the latest backup available in case
+the update fails, and you need to restore your system. If a Home Assistant
+update would fatally fail, you'd wouldn't have an interface to download or
+restore a backup from; hence this path is put in place.
+
+#### Backup on updating add-ons
+
+When updating add-ons, things are slightly different. The toggle exists here
+too, and is disabled by default. However, when you enable it, only a backup
+of your add-on and its data is created. 
+
+We automatically keep around the last previous version of the add-on, so you
+can always roll back to the previous version if needed.
+
+To prevent your system for filling up with old add-on backups, we automatically
+delete the previous versions of the add-on backup after a new one is created.
+
+### Improved filenames for backups
+
+One of the feedback we've seen a lot, is that the filenames created by
+backups stored on backup locations, have been very cryptic and hard to
+understand for humans.
+
+This feedback was fair, as the filename was a hash and originally not designed
+to be human readable. This method was not newly introduced last release, but it was made more apparent to us with the new backup system.
+
+So, this release, we've improved the filenames of the backups stored on your
+backup locations. The filename now includes the date and time the backup was
+created, making it easier to understand and identify the backup you are looking
+for, even outside of Home Assistant.
+
+### New locations
+
+This release also introduces new locations to store your backups, including [Google Drive](integrations/google_drive/) and [Microsoft OneDrive](/integrations/onedrive/)!
```

Comment:
Could we also mention the Synology DSM integration here? It's not because I want to highlight my self :see_no_evil: , but there are "only" these 3 new integrations implementing the backup-agent and we're very focused on improving the backup experience in this release

Reply:
Yes! Will add it!

Reply:
thx :)